### PR TITLE
Update RSWMap.java

### PR DESCRIPTION
**Pull request by RAFAELESPADA**

### CODE REVIEW

File: realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java [hunk 1]
```diff
@@ -192,7 +192,27 @@ public String forceStart(RSWPlayer p) {
             }
         }
     }
+public RSWMap BestArena() {
```
Comment (neziw) [inline realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java:195]: This is not C#, you should follow java naming convention, method names should be called in camelCase.

File: realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java [hunk 2]
```diff
@@ -192,7 +192,27 @@ public String forceStart(RSWPlayer p) {
...
+            }
+            if (pCount > mostPlayers) {
+                mostPopular = r;
+                mostPlayers = pCount;
+            }
 
+        }
+            return mostPopular;
+        }
+public boolean isBestArena() {
```
Comment (neziw) [inline realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java:213]: Add tab or spaces

File: realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java (hunk 1 above)
Comment (neziw) [reply realskywars-api/src/main/java/joserodpt/realskywars/api/map/RSWMap.java:195]: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

**Review by neziw — CHANGES_REQUESTED** [review]

**Comment by RAFAELESPADA** [comment]
You can "fix" that small things and commit my changed?

**Pull request merged**